docs/design/2021-06-23-spilled-unparallel-hashagg.md

```diff
@@ -45,7 +45,10 @@ In aggregate processing, memory increases when tuples are inserted into the hash
 
 ### Scenario Tests
 
-* In a scenario where aggregate executor use too many memory, this feature helps reduce memory usage and run the sql successfully.
+* When the unparallel-agg exceeds the memory quota, this feature helps reduce memory usage and run the sql successfully.
+* When the parallel-agg exceeds the memory quota, the SQL will be canceled before. After the agg-concurrency args are set to 1, the SQL can run successfully.
+* When the ndv of the data is low, the SQL contains distinct function will be canceled before. After the agg-concurrency args are set to 1, the SQL can run successfully.
+* When the ndv of the data is high, the SQL contains distinct function will be canceled before. After the agg-concurrency args are set to 1, the SQL can be canceled successfully if there is insufficient memory.
```

Comment:
Why do we need to set the concurrency-related args when there exists an aggregation function with the keyword `distinct`.

Reply:
Addressed